src/lib/translate.ts

```diff
@@ -155,10 +155,10 @@ export function ngettext(singular: string, plural: string, count: number) {
 
     /* If we don't have a ngettext translation entry at all, but
      * we do have some stand alone translations, use those */
-    if ((count !== 1 || !(singular in catalog)) && plural in catalog) {
+    if (count !== 1 && plural in catalog) {
```

Comment:
I'm not entirely sure if the way we were doing it was particularly useful, so I am fine with this change as it's "more correct" I reckon - but just curious, was this a noted problem somewhere or why the change?

Reply:
Granted, there was no issue before the updates in this PR.  However, it manifested itself like so:

<img width="486" alt="image" src="https://user-images.githubusercontent.com/25233703/103669858-dc76a480-4f46-11eb-8259-a839b8a3aa64.png">

The issue was that `ngettext` was detecting a _() translation of "%s days" but not "%s day".  I think it was borrowing the translation from `util.mktime()`, which has literals _("1 day") and _("%s days"). To be honest, I wasn't sure about the best way to fix this.

### Things I considered

- Check the language in ngettext `if (currentLanguage() != 'en' && ...)
- Add a translation in English
  - I see catalog is populated in English/debug, but I couldn't figure out how this works since there is no English project on translate.online-go.com
- Call interpolate("1 day", "%s days"
  - This would work, but would produce a warning
- Remove some of the "smarts" of `ngettext`

I went with the last option because it seemed the simplest and would prevent spooky-action-at-a-distance errors like the one I ran into.